commonutils/build.gradle

```diff
@@ -26,6 +26,7 @@ apply from: 'publish.gradle'
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
     def supportVersion = '24.2.1'
+    compile "com.android.support:support-v4:$supportVersion"
```

Comment:
avoid bringing in the whole library, only bring in the specific packages that are required https://developer.android.com/topic/libraries/support-library/packages.html